spec/models/user_spec.rb

```diff
@@ -426,4 +426,10 @@
       expect(user.events_registrations).to eq [@events_registration1, @events_registration2]
     end
   end
+
+  describe '.omniauth_providers' do
+    it 'contains providers' do
+      expect(User.omniauth_providers).to eq [:suse, :google, :facebook, :github]
```

Comment:
This is a partial test since it doesn't take into account the 'OSEM_ICHAIN_ENABLED' environment variable